private_repo/ansible/inventory/dev/Core/secrets.yml

```diff
@@ -8,42 +8,14 @@ core_vault_docker_registry_url: "change.docker.url"      # for docker hub "https
 core_vault_docker_registry_password: "change.docker.password"
 
-# Run the below command in shell
-# date +'%Y-%m-%dT%H:%m:%SZ' -d '+1 year'
-# sas_token=?`az storage account generate-sas --account-name "{{ azure_plugin_storage_account_name }}" --account-key "{{ azure_plugin_storage_account_key }}" --expiry $sas_expire_time --https-only --permissions acdlpruw --resource-types sco --services bfqt | xargs`
-# generate a sas for the blob for entire storage accout with write and read access
-sunbird_artifact_storage_account_sas: "change.azure.storage.account.sas"                        # SAS token value generated from above command
-sunbird_public_storage_account_sas: "change.azure.storage.account.sas"                          # SAS token value generated from above command
-sunbird_management_storage_account_sas: "change.azure.storage.account.sas"                      # SAS token value generated from above command
-
-sunbird_public_storage_account_key: "change.azure.storage.account.key"
-sunbird_private_storage_account_key: "change.azure.storage.account.key"
-sunbird_management_storage_account_key: "change.azure.storage.account.key"
-sunbird_artifact_storage_account_key: "{{ sunbird_management_storage_account_key }}"
-
-# Define the below if you are using Azure Cloud
-azure_public_storage_account_key: "{{ sunbird_public_storage_account_key }}"
-azure_private_storage_account_key: "{{ sunbird_private_storage_account_key }}"
-azure_management_storage_account_key: "{{ sunbird_management_storage_account_key }}"
-azure_artifact_storage_account_key: "{{ sunbird_artifact_storage_account_key }}"
-azure_public_storage_account_sas: "{{ sunbird_public_storage_account_sas }}"
-azure_management_storage_account_sas: "{{ sunbird_management_storage_account_sas }}"
-
-# Define the below if you are using AWS Cloud
-aws_management_bucket_access_key: ""
-aws_artifact_bucket_access_key: ""
-aws_public_bucket_access_key: ""
-aws_private_bucket_access_key: ""
-
-aws_management_bucket_secret_access_key: ""
-aws_artifact_bucket_secret_access_key: ""
-aws_public_bucket_secret_access_key: ""
-aws_private_bucket_secret_access_key: ""
-
-# Define the below if you are using Google Cloud
-gcp_storage_service_account_name: ""
-gcp_storage_key_file: ""  # gcloud service account key - refer: https://cloud.google.com/iam/docs/creating-managing-service-account-keys
-
+# Cloud Service Provider Secret Variables
+# If cloud_service_provider is aws--> secret access key
+# If cloud_service_provider is gcp--> service account key(copy private key from json file and update except for cloud_management_storage_secret)
+# If cloud_service_provider is azure--> storage account key
+cloud_public_storage_secret: ""     
+cloud_private_storage_secret: ""  
+cloud_management_storage_secret: "" # If cloud_service_provider is gcp --> copy full json file content and update only for cloud_management_storage_secret as its used for operational tasks such as backups
```

Comment:
Mention this var before the `cloud_private_storage_secret` var. Give examples and reference to previous vars where required.